mitodl/open-discussions
# add/remove items from userlists

#### Pre-Flight checklist


#### What are the relevant tickets?
#3819 

#### What's this PR do?
This PR:
- Adds 'add to userlist' functionality on resource cards
- renames the old resource card to LearningResourceCardTemplate. This is display only and does not have associated user interactions. Adds a new LearningResourceCard that triggers the drawer and 'add to userlist' dialog.

#### How should this be manually tested?
1. As an anonymous user, visit https://od.odl.local:8063/infinite/search. Resource cards should **not** have a bookmark icon for adding to list.
2. As a logged-in user, visit https://od.odl.local:8063/infinite/search. Resource cards **should** have a bookmark icon for adding to list.
    - click the bookmark icon
    - add/remove a resource to a list
    - also try creating a new list from the 'add to list' dialog.
3. Visit https://od.odl.local:8063/infinite/lists, and check that your items have been added to the list.

#### Screenshots
![Uploading Screenshot 2023-03-21 at 2.12.25 PM.png…]()
